doc/release-notes.md

```diff
@@ -47,8 +47,9 @@ processing the entire blockchain.
 Compatibility
 ==============
 
-Bitcoin Core is extensively tested on multiple operating systems using
-the Linux kernel, macOS 10.10+, and Windows 7 and newer (Windows XP is not supported).
+Bitcoin Core is extensively tested and supported on operating systems using
```

Comment:
"extensively tested and supported" reads a bit strange

Reply:
Oh yeah that's bad I'll switch the 2